client/src/app.jsx

```diff
@@ -32,12 +32,15 @@ class App extends React.Component {
   }
 
   componentDidMount() {
-    this.getStateData();
+    const urlSearchParams = new URLSearchParams(window.location.search);
```

Comment:
Learned a bunch of new things in just these few lines, I'd never seen this syntax before